src/main/webapp/dashboard.html

```diff
@@ -114,16 +137,21 @@ <h4 class="card-title"></i><a>Today's Tasks</a></h4>
                     </div>
                 </div>
             </div>
-            <div id="draggable-1" class="card">
+            <div id="draggable-1" class="card" style="width: 460px;">
                 <h5 class="card-header primary-color white-text">#Tag</h5>
-                <div class="card-body">
-                    <form id="tag-form" action="/dashboard/calendar" method="POST" class="hidden">
+                <div class="card-body-tag">
+                    <form id="table-form" action="/dashboard/calendar" method="POST" class="hidden" name="Form1">

```

Comment:
You shouldn't use the same id as mine because id's definition is unique for that element